spring-integration-core/src/main/java/org/springframework/integration/config/AbstractStandardMessageHandlerFactoryBean.java

```diff
@@ -129,22 +130,22 @@ private void checkReuse(AbstractMessageProducingHandler replyHandler) {
 	/**
 	 * Subclasses must implement this method to create the MessageHandler.
 	 */
-	abstract MessageHandler createMethodInvokingHandler(Object targetObject, String targetMethodName);
+	protected abstract MessageHandler createMethodInvokingHandler(Object targetObject, String targetMethodName);
```

Comment:
```
D:\SpringIO\spring-integration\spring-integration-core\src\main\java\org\springframework\integration\config\AbstractStandardMessageHandlerFactoryBean.java:133: warning: no @param for targetObject
        protected abstract MessageHandler createMethodInvokingHandler(Object targetObject, String targetMethodName);
```
Strange for me from the `protected`, but will fix on merge.
This and others with similar vulnerability.